<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformationWindow</class>
 <widget class="QMainWindow" name="InformationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>31</y>
      <width>721</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>Select Information To Display</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Display NFL Stadiums and Corresponding Team Names</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Display List of American Conference Teams</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Display List of National Conference Teams</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Display List of Open Roof Stadiums</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Display Star Players and Corresponding Teams</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Display NFL Stadium Surface Types</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Display NFL Stadium Seating Capacities</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QTableView" name="Table"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Select a Team to Display Its Information</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Display All Team</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
